GameReadyValidator/src/main/java/pl/przemyslawjakubowski/tajgertim/gamereadyvalidator/gamevalidation/Confirmations.java

```diff
@@ -0,0 +1,45 @@
+package pl.przemyslawjakubowski.tajgertim.gamereadyvalidator.gamevalidation;
+
+import pl.jakubowskiprzemyslaw.tajgertim.models.confirmation.Confirmation;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+class Confirmations {
+
+    private Set<Class> confirmations;
+
+    Confirmations() {
+        initConfirmations();
+    }
+
+    private void initConfirmations() {
+        confirmations = new HashSet<>();
+    }
+
+    void addConfirmation(Confirmation confirmation) {
+        confirmations.add(confirmation.getClass());
```

Comment:
what about null

Reply:
issue fixed by changing type of stored elements